Libraries/LibWeb/Cookie/ParsedCookie.cpp

```diff
@@ -95,7 +95,7 @@ Optional<ParsedCookie> parse_cookie(URL::URL const& url, StringView cookie_strin
         return {};
 
     // 6. The cookie-name is the name string, and the cookie-value is the value string.
-    ParsedCookie parsed_cookie { MUST(String::from_utf8(name)), MUST(String::from_utf8(value)) };
+    ParsedCookie parsed_cookie { String::from_utf8_with_replacement_character(name), String::from_utf8_with_replacement_character(value) };
```

Comment:
Just wrapping my head around the problem - `name` and `value`, when originating from a response header, could potentially contain any garbage bytes? Or are they specific to some encoding?

If the former - seems like the fix would be to make those two fields `ByteString`.
If the latter - this seems okay, but I'm not finding spec text that supports this (I can totally be missing something!)

Is it possible for any of the attribute values contained in `unparsed_attributes` to contain invalid bytes? Those also seem to blindly `MUST(from_utf8())` below. For example, the domain and host attributes don't seem to check for valid UTF-8 either.